.github/workflows/pfx_to_file.yml

```diff
@@ -1,22 +1,42 @@
 # Workflow to test secrets, first version empty in order to be able to run it from branch
-
 name: PFX To File
 
-# Controls when the workflow will run
 on:
-  # Allows you to run this workflow manually from the Actions tab
-  workflow_dispatch:
+    workflow_dispatch:
 
-# A workflow run is made up of one or more jobs that can run sequentially or in parallel
 jobs:
```

Comment:
Add the property name for each job and step.
- [ ] name property

Reply:
Done